firestore-counter/extension.yaml

```diff
@@ -47,7 +47,9 @@ resources:
       sourceDirectory: .
       location: ${LOCATION}
       maxInstances: 1
-      httpsTrigger: {}
+      eventTrigger:
+        eventType: google.pubsub.topic.publish
+        resource: projects/${PROJECT_ID}/topics/firestore-counter
```

Comment:
Can you make the topic the same as the ID of the instance? It is available as ${MOD_INSTANCE_ID}. That way, if someone installs this extension more than once, it will still work properly. 

This also needs to be changed in the POSTINSTALL. 

Meanwhile, I'm going to change the backend to also populate EXT_INSTANCE_ID, since the product is no longer called Mods. Hopefully, that'll be in prod before we release this update, then you can change MOD_INSTANCE_ID with EXT_INSTANCE_ID before merging the PR.

Reply:
Ah this makes more sense, pushed up a change for this - thanks!

(will update to `EXT_INSTANCE_ID` when available)